test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua

```diff
@@ -0,0 +1,141 @@
+end
+
+Preconditions:BackupFile("sdl_preloaded_pt.json")
+testCasesForPolicyTable:Precondition_updatePolicy_By_overwriting_preloaded_pt("files/DeviceGroupInPreconsented_preloadedPT.json")
+
+function Test:Precondition_ConnectDevice()
+	commonTestCases:DelayedExp(2000)
+	self:connectMobile()
+	EXPECT_HMICALL("BasicCommunication.UpdateDeviceList",
+					{
+						deviceList = {
+							{
+								id = config.deviceMAC,
+								isSDLAllowed = true,
+								name = "127.0.0.1",
+								transportType = "WIFI"
+							}
+						}
+					}
+	):Do(function(_,data)
+		self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+	end)
+	:Times(AtLeast(1))
+end
+
+function Test:Precondition_RegisterApp()
+	commonTestCases:DelayedExp(3000)
+	self.mobileSession = mobile_session.MobileSession(self, self.mobileConnection)
+	self.mobileSession:StartService(7)
+	:Do(function()
+		local correlationId = self.mobileSession:SendRPC("RegisterAppInterface", config.application1.registerAppInterfaceParams)
+		EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered")
+		:Do(function(_,data)
+			self.HMIAppID = data.params.application.appID
+		end)
+		self.mobileSession:ExpectResponse(correlationId, { success = true, resultCode = "SUCCESS" })
+		self.mobileSession:ExpectNotification("OnHMIStatus", {hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN"})
+	end)
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+function Test:Step1_PTU_lack_of_user_consent_prompt()
+	local RequestIdGetURLS = self.hmiConnection:SendRequest("SDL.GetURLS", { service = 7 })
+	EXPECT_HMIRESPONSE(RequestIdGetURLS,{result = {code = 0, method = "SDL.GetURLS", urls = {{url = "http://policies.telematics.ford.com/api/policies"}}}})
+	:Do(function()
+		self.hmiConnection:SendNotification("BasicCommunication.OnSystemRequest",
+			{
+				requestType = "PROPRIETARY",
+				fileName = "filename"
+			}
+		)
+		EXPECT_NOTIFICATION("OnSystemRequest", { requestType = "PROPRIETARY" })
+		:Do(function()
+			local CorIdSystemRequest = self.mobileSession:SendRPC("SystemRequest",
+				{
+					fileName = "PolicyTableUpdate",
+					requestType = "PROPRIETARY"
+				}, "files/PTU_LackOfUserConsentPromptInGroup.json")
+			local systemRequestId
+			EXPECT_HMICALL("BasicCommunication.SystemRequest")
+			:Do(function(_,data)
+				systemRequestId = data.id
+				self.hmiConnection:SendNotification("SDL.OnReceivedPolicyUpdate",
+				{
+					policyfile = "/tmp/fs/mp/images/ivsu_cache/PolicyTableUpdate"
+			})
+			local function to_run()
+				self.hmiConnection:SendResponse(systemRequestId,"BasicCommunication.SystemRequest", "SUCCESS", {})
+			end			
+			RUN_AFTER(to_run, 500)
+		end)
+		EXPECT_HMINOTIFICATION("SDL.OnAppPermissionChanged", {appID = self.HMIAppID,  appPermissionsConsentNeeded = false})
+		self.mobileSession:ExpectResponse(CorIdSystemRequest, {success = true, resultCode = "SUCCESS"})
+		end)
+	end)
+end
+
+function Test:Step2_Activate_app_no_consent()
```

Comment:
Please see comment above. All tests in this section should follow the convention prefix TestStep_ + description